lib/action_controller/serialization.rb

```diff
@@ -8,8 +8,10 @@ module Serialization
     include ActionController::Renderers
 
     module ClassMethods
-      def serialization_scope(scope)
-        self._serialization_scope = scope
+      def serialization_scope(scope, opts = {})
+        before_action(opts.slice(:only, :except, :if, :unless)) do
```

Comment:
I agree to your suggestion @bf4 about moving this to userland, created #1824 with added documentation.